File: C/program5.c
#include<stdio.h>

void DisplayClass(float fPerc)
{
    if((fPerc < 0.0f) || (fPerc > 100.0f))
    {
        printf("Unable to proceeed as input is invalid\n");
        return;
    }

    if((fPerc >= 0.0f) && (fPerc < 35.0f))
    {
        printf("You are fail\n");
    }
    else if((fPerc >= 35.0f) && (fPerc < 50.0f))
    {
        printf("Pass class\n");
    }
    else if((fPerc >= 50.0f) && (fPerc < 60.0f))
    {
        printf("Second class\n");
    }
    else if((fPerc >= 60.0f) && (fPerc < 70.0f))
    {
        printf("First class\n");
    }
    else if((fPerc >= 70.0f) && (fPerc <= 100.0f))
    {
        printf("First clas with distinction\n");
    }
}

int main()
{
    float fValue = 0.0f;

    printf("Please enter your percentage : \n");
    scanf("%f",&fValue);

    DisplayClass(fValue);

    return 0;
}

File: Python/program5.py
def Display(fPerc):
    if fPerc < 0.0 or fPerc > 100.0:
        print("Unable to proceed as input is invalid")
        return

    if 0.0 <= fPerc < 35.0:
        print("You are fail")
    elif 35.0 <= fPerc < 50.0:
        print("Pass class")
    elif 50.0 <= fPerc < 60.0:
        print("Second class")
    elif 60.0 <= fPerc < 70.0:
        print("First class") 
    elif 70.0 <= fPerc <= 100.0:
        print("First class with distinction")


def main():
    print("Plase Enter the Percentage:")
    fValue = float(input())
    Display(fValue)


if __name__ == "__main__":
    main()
